Add FINPLAN

naomiyct opened the pull request:
https://github.com/FINPLAN-Model/FINPLAN-UI

The project is:

- [x] Active
- [x]  Documented
- [x]  Licensed with an open source license
- [x]  Shows usage from external parties
- [x]  Directly targets environmental sustainability

Find more details in the [Contribution Guide](https://opensustain.tech/contributing/).

All listed projects on [OpenSustain.tech](https://opensustain.tech/) will be supported via multiple community services:

1. Issues labeled as Good First Issue will be visible on [ClimateTriage.com](https://climatetriage.com/). This is a great way to welcome new community members to your project.
2. All new projects listed will be posted on our [Mastodon](https://mastodon.social/@opensustaintech) and [Bluesky](https://bsky.app/profile/opensustaintech.bsky.social) channel.
3. Listed projects that provide a donation option can by nominated via [OpenClimate.fund](https://openclimate.fund/). All projects currently supported can be found [here](https://opensustain.tech/spreadsheet/).

Insert a project you want to nominate for a microgrant here: If no project is nominated by the pull request author, developers mentioned in the comments can nominate a project.

Ly0n commented:
Hey @naomiyct, the code repository behind this project was active 2 years ago. Also the code shows not usage indicators from external party. I will put the project Under Observation. Do you need help to promote this project or to find external contributor / users? We can help here with the community.

naomiyct commented:
Hi @Ly0n, thanks for your comment! On the repo README file, you can see the number of downloads. Also, the online learning for this tool directs users to this repo (see Hands-on Exercise 1 here: www.open.edu/openlearncreate/pluginfile.php/880164/mod_resource/content/8/FINPLAN_Hands-on_1.pdf), is this sufficient evidence? There are also some FINPLAN case studies available online, and we are currently trying to collate them all. Here are some examples: https://nucet.pensoft.net/article/54003/ & https://zenodo.org/records/10600780
The code repo hasn't been active since it was initially uploaded as there haven't been any updates - although there are plans to do so soon. And yes, promoting the model and finding external contributors/users will be great :)

naomiyct commented:
Hi @Ly0n ,
I just wanted to check in on this! Is the information above sufficient?
We are also conducting capacity building with FINPLAN, with one event currently taking place at UNECA. See here: https://www.uneca.org/eca-events/EMP-A-2025 
I have also updated the pull request and moved FINPLAN to Sustainable Investment, where I think it fits better.

Ly0n commented:
Hi @naomiyct Many thanks for the additional information. The use is now definitely proven.
